modules/spring-config/src/main/java/org/mule/runtime/config/spring/ModuleDelegatingEntityResolver.java

```diff
@@ -93,7 +95,9 @@ public InputSource resolveEntity(String publicId, String systemId) throws SAXExc
     if (inputSource == null) {
       inputSource = springEntityResolver.resolveEntity(publicId, systemId);
     }
-
+    if (inputSource == null && !extensions.isEmpty()) {
```

Comment:
Add comment explaining that the code is sometimes called before loading extensions and after, so we have to allow a repetition in that case.